artemis-commons/src/main/java/org/apache/activemq/artemis/api/core/QueueAttributes.java

```diff
@@ -103,6 +105,8 @@ public void set(String key, String value) {
             setAutoDeleteMessageCount(Long.valueOf(value));
          } else if (key.equals(RING_SIZE)) {
             setRingSize(Long.valueOf(value));
+         } else if (key.equals(ENABLED)) {
+            setRingSize(Long.valueOf(value));
          }
```

Comment:
Shouldn't this be `setEnabled(Boolean.valueOf(value));`?

Reply:
yes good catch

Reply:
fixed